src/hooks/useProfileRedirect.ts

```diff
@@ -0,0 +1,96 @@
+import { useEffect } from 'react';
+
+type ProfileStatus = 'activated' | 'created' | 'deleted' | 'error' | 'profile-completed' | 'social-handle-provided' | 'tos-accepted';
+
+interface ProfileStatusResponse {
+	id: string;
+	name: string;
+	access: string;
+	avatar: string | null;
+	status: ProfileStatus;
+}
+
+const REDIRECT_PATHS = {
+	signIn: '/pages/sign-in/',
+	// TODO: Determine how to redirect uses who created account but failed to activate immediately
```

Comment:
They should be sent to the sign-in page. Until they confirm their email, they will be locked out of the platform.